src/cc65/compile.c

```diff
@@ -300,7 +299,7 @@ void Compile (const char* FileName)
     struct tm*  TM;
 
     /* Since strftime is locale dependent, we need the abbreviated month names
-    ** in english.
+    ** in English.
```

Comment:
So ... according to Wikipedia, all of my english teachers -- oops, English teachers -- were wrong.  OK.